frontend/packages/ceph-storage-plugin/src/constants/queries.ts

```diff
@@ -79,7 +81,10 @@ export const CAPACITY_BREAKDOWN_QUERIES = {
   [StorageDashboardQuery.PODS_BY_USED]:
     'sum((kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"})) by (pod,namespace)',
   [StorageDashboardQuery.CEPH_CAPACITY_TOTAL]: 'ceph_cluster_total_bytes',
-  [StorageDashboardQuery.CEPH_CAPACITY_USED]: 'ceph_cluster_total_used_bytes',
+  [StorageDashboardQuery.CEPH_CAPACITY_USED]:
+    'sum(kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"}))',
+  [StorageDashboardQuery.CEPH_CAPACITY_AVAILABLE]:
+    'max(ceph_pool_max_avail * on (pool_id) group_left(name)ceph_pool_metadata{name=~"(.*file.*)|(.*block.*)"})',
```

Comment:
@anmolsachan Please review the query here.

Reply:
lgtm

Reply:
lol! lgtm on your written query :D @anmolsachan